### PR TITLE
Admin noah

### PR DESCRIPTION
**Pull request by noahhauke22**

### CODE REVIEW

**Review by LucRome — APPROVED** [review]

File: eCourse_backend/courses/views.py [hunk 1]
```diff
@@ -26,26 +26,52 @@ def course_student_list_iframe(request, name, page=1):
...
+    paginator = Paginator(courses, 10)
+    page_obj = paginator.get_page(page)
+
+    context = {
+        'page_obj': page_obj,
+    }   
+    return render(request, 'courses/overview.html', context)
+
 @login_required
 @permission_required('courses.create_course', raise_exception=True)
```
Comment (phie73) [inline eCourse_backend/courses/views.py]: i don't think we need the permission create_course for the overview function. As it is written right now only lecturers and officers can view this. But this is something students also should see

**Review by phie73 — CHANGES_REQUESTED** [review]
change permission for view course (rm create_course) (see comment in code)

File: eCourse_backend/courses/views.py [hunk 2]
```diff
@@ -26,26 +26,52 @@ def course_student_list_iframe(request, name, page=1):
...
     if request.method == 'GET':
-        courses = Course.objects.all()
-        return render(request, 'courses/overview.html', {'courses': courses})
+        course = get_object_or_404(Course, pk = id)
+        lecturer = course.lecturer_id
+        # print('lecturer', lecturer)
+        students = list()
+        for student in course.student.all():
+            # print(student.id)
+            students.append(student.id)
```
Comment (LucRome) [inline eCourse_backend/courses/views.py:73]: Wouldn't `students = course.student.all()` work?

**Review by LucRome — CHANGES_REQUESTED** [review]

File: eCourse_backend/courses/views.py (hunk 1 above)
Comment (LucRome) [reply eCourse_backend/courses/views.py]: This View is from the backend and isn't used by the frontend, so it shouldn't matter for this PR

Comment (phie73) [reply eCourse_backend/courses/views.py]: yes you are right but i think is is something we can fix by removing one line of code and that is in my oppinion something that can be done befor merging to the master branch

**Review by phie73 — APPROVED** [review]
thank you

**Pull request closed without merging**